website/docs/rules/index.mdx

```diff
@@ -712,6 +712,15 @@ Rules are grouped by category to help you understand their purpose. Each rule ha
   methods name argument.
 </RuleEntry>
 
+<RuleEntry
+  name="prefer-provide-intl-description"
+  type="intl"
+  severity="warning"
+  version="5.5.0"
+>
+  Warns when a `Intl` method invocation without description.
```

Comment:
Warns when a method is invoked without a description.

Reply:
fixed